src/lib/factories.js

```diff
@@ -46,7 +46,7 @@ export function createShorthand(Component, mapValueToProps, val, defaultProps =
   } else if (_.isPlainObject(val)) {
     type = 'props'
     usersProps = val
-  } else if (_.isString(val) || _.isNumber(val)) {
+  } else if (_.isString(val) || _.isNumber(val) || _.isArray(val)) {
```

Comment:
Cool, I think we're seeing the same split in types of shorthand.  This is what I ran into [here](https://github.com/TechnologyAdvice/stardust/pull/452#issuecomment-243614062) where I reference `customPropTypes.shorthandSingle` and `customPropTypes.shorthandMultiple`.  I like collection a lot better than multiple though 😄.

So, I think we have 2 confirmed types of shorthand then.  Both handle a props objects and elements identically.  The difference is in what other values they accept and how they `mapValueToProps`:

1. Single item factory (Icon, etc.) also handles **string/number** mapping them to props.
1. Collection factory (Menu, etc.) also handles **arrays** mapping them to a single collection prop (items, cells, etc).

There is a 3rd group we can deal with later, since I think they need API redesign. Breadcrumb, Accordion, and possibly Feed, which take an array but do not map it to props.  Instead, they create children in a special way, usually due to markup constraints.  We'll save that for another day and time.

I'm working on the Feed updates and shorthand propTypes as I find time.  Won't hold up this PR for that work.  I can circle back and update when my work there is ready.